src/scope.rs

```diff
@@ -115,20 +127,30 @@ pub fn analyze<'a>(files: &'a Map<PathBuf, Parsed>, config: &ConfigFile, emitter
 
             already_analyzed.insert(UniCase::new(&usage.name));
 
-            match scope.search(&UniCase::new(&usage.name)) {
-                None => {
+            let result = scope.search(&usage.name);
+            match result {
+                (None, _) => {
                     usage.location.in_file(&parsed.original_path)
                         .lint(Lint::UnknownFunctions, format!("Not in scope: {}", usage.name))
                         .what(usage.name.clone())
                         .emit(emitter);
                 }
-                Some(Found::Indirect) => {
+                (Some(Found::Indirect), _) => {
                     usage.location.in_file(&parsed.original_path)
                         .lint(Lint::IndirectImports, format!("Indirectly imported: {}", usage.name))
                         .what(usage.name.clone())
                         .emit(emitter);
                 }
-                _ => (),
+                _ => ()
+            }
+            match result {
```

Comment:
as above, I don't see a nicer way to do this by pattern matching than to pattern match twice on a tuple